lib/features/dashboard/home/home_view.dart

```diff
@@ -0,0 +1,288 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:intl/intl.dart';
+import 'package:live_score/core/extensions/localization/app_localizations_context.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/core/ui/dashboard/home/home_appbar.dart';
+import 'package:live_score/core/ui/dashboard/home/home_calendar.dart';
+import 'package:live_score/core/ui/dashboard/home/home_favorites_card.dart';
+import 'package:live_score/core/ui/dashboard/home/home_live_card.dart';
+import 'package:live_score/core/ui/dashboard/home/home_score_card.dart';
+import 'package:live_score/core/ui/dashboard/home/home_tabs.dart';
+import 'package:live_score/core/ui/dashboard/home/home_upcoming_card.dart';
+import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
+
+class HomeView extends StatelessWidget {
+  const HomeView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final String currentDate = DateFormat('yyyy-MM-dd').format(DateTime.now());
+    final cubit = context.read<DashboardCubit>();
+    cubit.fetchFixtures(currentDate);
```

Comment:
This method shouldn't be called here, it will be called again and again with each rebuild